README.md

```diff
@@ -0,0 +1,118 @@
+# simba-starter
+
+A [lion](https://lion-web.netlify.app/) based design system as a starter kit
+for [Backlight](https://backlight.dev/), an all-in-one Design System platform,
+design inspired by [TailwindCSS](https://tailwindcss.com/).
+
+## [🔗 See it in action](https://backlight.dev/preview/5vtJtbY04aoD1dGKcsu1)
+
+Click the "Edit" button to also see the source code in the Backlight editor.
+
+## Features
+
+- 🍴 Easy to fork and create your own variation using [Backlight](https://backlight.dev/)
+- 📦 Available on NPM to extend or use directly
+- 🌙 Darkmode out of the box
+- 🦄 Theming capabilities, choose any color from Tailwind palette out of the box, with room for more!
+
+## Usage
+
+Importing a component to extend:
+
+```js
+import { SimbaButton } from '@divriots/starter-simba/button';
+
+class NalaButton extends SimbaButton {}
+```
+
+Using a component directly in your application
+
+```html
+<head>
+  <script type="module" src="@divriots/starter-simba/button/simba-button.js"></script>
+</head> 
+<body>
+  <simba-button>Submit</simba-button>
+</body>
+```
+
+or in JS using lit template:
+
+```js
+import { html } from '@divriots/starter-simba/core';
+import '@divriots/starter-simba/button/simba-button.js';
+
+export const templ = html`<simba-button>Submit</simba-button>`;
+```
+
+### Backlight
+
+This is the recommended way, as backlight gives you an all-in-one Design System platform.
+
+In order to create your own project from this starter kit using Backlight, go to the [Backlight website](https://backlight.dev/) and sign up, request early access, or log in if you already have an account.
+
+Make sure you have a Backlight Workspace, then go to [simba-starter on backlight](https://backlight.dev/edit/5vtJtbY04aoD1dGKcsu1) and click the **Duplicate** button in the top right corner, this will create a project based on this starter-kit for you.
+
+### NPM
```

Comment:
Maybe I would put the use of the "default" starterkit from npm as a separate .md file and link it here.
So we can really focus on the happy flow with backlight but explain there is an alternative.

Reply:
Yep makes sense, done in a separate MD now